sklearn/linear_model/tests/test_common.py

```diff
@@ -161,6 +163,7 @@ def test_balance_property(model, with_sample_weight, global_random_seed):
 
 @pytest.mark.filterwarnings("ignore:The default of 'normalize'")
 @pytest.mark.filterwarnings("ignore:lbfgs failed to converge")
+@pytest.mark.filterwarnings("ignore:A column-vector y was passed when a 1d array.*")
```

Comment:
Just a question about why this was needed?

Reply:
I don’t know. I usually want tests running without warnings. I guess it has nothing to do with the main topic of this PR.